skops/cli/_convert.py

```diff
@@ -0,0 +1,109 @@
+from __future__ import annotations
+
+import argparse
+import logging
+import os
+import pathlib
+import pickle
+from typing import Optional
+
+from skops.cli._utils import get_log_level
+from skops.io import dumps, get_untrusted_types
+
+
+def _convert_file(input_file: os.PathLike, output_file: os.PathLike):
+    """
+    Function that is called by ``skops convert`` entrypoint.
+
+    Loads a pickle model from the input path, converts to skops format, and saves to
+    output file.
+
+    Parameters
+    ----------
+    input_file : os.PathLike
+        Path of input .pkl model to load.
+
+    output_file : os.PathLike
+        Path to save .skops model to.
+
+    """
+    model_name = pathlib.Path(input_file).stem
+
+    logging.debug(f"Converting {model_name}")
+
+    with open(input_file, "rb") as f:
+        obj = pickle.load(f)
+    skops_dump = dumps(obj)
+
+    untrusted_types = get_untrusted_types(data=skops_dump)
+
+    if not untrusted_types:
+        logging.info(f"No unknown types found in {model_name}.")
+    else:
+        untrusted_str = ", ".join(untrusted_types)
+
+        logging.warning(
+            "Unknown Types Detected! "
```

Comment:
Probably not, though I thought it might be useful to see an immediate descriptor at the start of the log, particularly if it's a long log. Will remove though.